mix.exs

```diff
@@ -5,7 +5,7 @@ defmodule EpochElixir.Mixfile do
     [app: :epoch_elixir,
      apps_path: "apps",
      version: "0.1.0",
-     elixir: "~> 1.5.1",
+     elixir: "> 1.5.1",
```

Comment:
This is not Elixir `1.6.0` support. My understanding is this says "only use a version of elixir greater than `1.5.1`". If you have 1.5.2 installed, it will use that. See: https://hexdocs.pm/elixir/Version.html

Reply:
Yes correct it should be >=